README.md

```diff
@@ -204,6 +207,35 @@ reporters.insert(CustomReporter.self, at: 1)
 let report = DiagnosticsReporter.create(using: reporters)
 ```
 
+## Smart Insights
+![](Assets/smart-insights.png)
+By default, standard Smart Insights are provided:
+
+- `UpdateAvailableInsight` using your bundle identifier to fetch the latest available app version. An insight will be shown whether an update is available to the user or not.
```

Comment:
Suggestion:

```suggestion
- `UpdateAvailableInsight` uses your bundle identifier to fetch the latest available app version. An insight will be shown whether an update is available to the user or not.
```

To me that reads better.

Reply:
Agreed!